src/dispatchers.ts

```diff
@@ -229,6 +230,69 @@ export async function handleAnnounce(
     await addToList(ctx.data.db, ['inbox'], announce.id.href);
 }
 
+export async function handleLike(
+    ctx: Context<ContextData>,
+    like: Like,
+) {
+    console.log('Handling Like');
+
+    // Validate like
+    if (!like.id) {
+        console.log('Invalid Like - no id');
+        return;
+    }
+
+    if (!like.objectId) {
+        console.log('Invalid Like - no object id');
+        return;
+    }
+
+    // Validate sender
+    const sender = await like.getActor(ctx);
+
+    if (sender === null || sender.id === null) {
+        console.log('Sender missing, exit early');
+        return;
+    }
+
+    // Lookup liked object - If not found in globalDb, perform network lookup
+    let object = null;
+    let existing = await ctx.data.globaldb.get([like.objectId.href]) ?? null;
+
+    if (!existing) {
+        console.log('Object not found in globalDb, performing network lookup');
+
+        object = await lookupObject(like.objectId);
```

Comment:
This can be simplified:

```suggestion
        object = await like.getObject();
```

Reply:
Thanks @dahlia thats updated in https://github.com/TryGhost/ActivityPub/pull/33/commits/81816ea00bb5251a11eebd22218e955839443524